test/e2e/allocation.new.test.js

```diff
@@ -0,0 +1,40 @@
+import { pmuUser } from './_roles'
+import { newAllocation } from './_routes'
+import { allocationJourney } from './pages/'
+
+fixture('New PMU allocation').beforeEach(async t => {
+  await t.useRole(pmuUser).navigateTo(newAllocation)
+})
+test('Create allocation and verify the result', async t => {
+  await allocationJourney.createAllocation()
+  const currentUrl = await allocationJourney.getCurrentUrl()
+  const allocationId = currentUrl.match(
+    new RegExp(allocationJourney.uuidRegex)
+  )[0]
+  await t.navigateTo(`/allocation/${allocationId}`)
+  ;['summary', 'meta'].forEach(async section => {
+    for (const o of allocationJourney.allocationViewPage.nodes[section].keys) {
+      const textExists = allocationJourney.allocationViewPage.getDlDefinitionByKey(
+        allocationJourney.allocationViewPage.nodes[section].selector,
+        o
+      )
+      await t.expect(textExists).ok()
+    }
+  })
+})
+test('Check validation errors on allocation details page', async t => {
+  await allocationJourney.submitForm()
+  for (const item of allocationJourney.allocationDetailsPage.nodes
```

Comment:
I think this is the kind of thing that would've been nice to extract to the main page object. To have a shared method to check the error messages on a form page. 

You're already doing it twice in this file and it's something we can definitely use elsewhere to improve our test suite.

Reply:
Let's leave this for a future improvement actually.